osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -42,7 +42,7 @@ public DrawableHitCircle(OsuHitObject h) : base(h)
 
                         Judgement.PositionOffset = Vector2.Zero; //todo: set to correct value
                         UpdateJudgement(true);
-                        return true;
+                        return !(Parent is DrawableSlider);

```

Comment:
Fair call, I'll revert the change in `DrawableHitCircle` and try to do something to check the actions in `SliderBall.Update()` then.